src/repositories/critter-repository.ts

```diff
@@ -377,7 +477,7 @@ export class CritterRepository extends Repository {
         where: {
           critter_id: critterId
         },
-        data: critterData,
+        data: { ...critterData },
```

Comment:
nitpick: any reason to spread the critterData here? Should just result in the same thing as assigning it directly?

Same on line 505 below.